waku/v2/protocol/waku_rln_relay/waku_rln_relay_metrics.nim

```diff
@@ -0,0 +1,17 @@
+import
+    metrics
+
+export metrics
+
+declarePublicCounter(waku_rln_messages, "number of messages published on the rln content topic")
+declarePublicCounter(waku_rln_spam_messages, "number of spam messages detected")
+declarePublicCounter(waku_rln_invalid_messages, "number of invalid messages detected", ["type"])
+# This metric will be useful in detecting the index of the root in the acceptable window of roots
+declarePublicCounter(waku_rln_valid_messages, "number of valid messages with their roots tracked", ["index"])
+declarePublicCounter(waku_rln_errors, "number of errors detected while operating the rln relay", ["type"])
+declarePublicCounter(waku_rln_proof_verification, "number of times the rln proofs are verified")
+
+# Timing metrics
+declarePublicHistogram(waku_rln_proof_verification_time, "time taken to verify a proof")
+declarePublicGauge(waku_rln_relay_mounting_time, "time taken to mount the waku rln relay")
```

Comment:
Why not use the same collection type for all the time values i.e., right now, the verification time and proof generation are of `histogram` type and the mounting time is a `Gauge` type,  why are they different?
Also, have you considered using [summary](https://github.com/status-im/nim-metrics#summary) instead for calculating the average time? Does histogram calculate average? (I guess not)

Reply:
`waku_rln_relay_mounting_time` is a gauge since mounting happens only once in the lifetime of the process. The others run frequently throughout the lifetime of the process

Reply:
However, since nodes are allowed to be online sporadically, it could be useful to track it in a histogram instead. Thanks for the tip. 
Addressed in [c30d7aa](https://github.com/status-im/nwaku/pull/1185/commits/c30d7aa3ea2946bd835da194630fd7692745192f)